internal/gengapic/helpers.go

```diff
@@ -122,6 +122,19 @@ func hasMethod(service *descriptor.ServiceDescriptorProto, method string) bool {
 	return false
 }
 
+// hasRESTMethod reports if there is at least one RPC on the Service that
+// has a gRPC-HTTP transcoding, or REST, annotation on it.
+func hasRESTMethod(service *descriptor.ServiceDescriptorProto) bool {
+	for _, m := range service.GetMethod() {
+		eHTTP := proto.GetExtension(m.GetOptions(), annotations.E_Http)
+		if h := eHTTP.(*annotations.HttpRule); h.GetPattern() != nil {
```

Comment:
No, because `proto.GetExtension` will always return a `HttpRule`. If the annotation is unset, a default one is returned. If the annotation is present, then it will be a `HttpRule`.